indico/modules/events/registration/forms.py

```diff
@@ -278,12 +279,31 @@ def validate_users_field(self, field):
                                   .format(emails=', '.join(sorted(existing))))
 
 
+class CSVFieldDelimiter(RichIntEnum):
+    __titles__ = [_('Comma'), _('Semicolon'), _('Tab'), _('Space')]
+    __delimiters__ = {
+        0: ',',
+        1: ';',
+        2: '\t',
+        3: ' ',
+    }
+
+    comma = 0
+    semicolon = 1
+    tab = 2
+    space = 3
+
+    def char(self):
```

Comment:
`@property` is your friend here (and I'd probably call it `delimiter` instead of `char`)

Reply:
Both addressed 👍